src/summariser.py

```diff
@@ -36,15 +66,19 @@ def cohere_summarise(message: str) -> NonStreamedChatResponse:
 
 
 def summarise_webpage(url: str) -> str:
-    # html_text = get_html_from_url(url)
-    cohere_response: NonStreamedChatResponse = cohere_summarise(url)
+    html_text = langchain_webcontent(url)
+    cohere_response: NonStreamedChatResponse = cohere_summarise(html_text)
     text: str = cohere_response.text
     return text
 
 
 if __name__ == "__main__":
```

Comment:
Strongly recommend you look into Fire by Google https://github.com/google/python-fire

Reply:
why would i need fire for a command line argument in here? This is just a function that i am using in this web app?

Reply:
see Issue: https://github.com/tommylees112/whatsappLLM/issues/5